### PR TITLE
Cp 170 set extension version in dom

### PR DESCRIPTION
**Pull request by Hatscat**
[Specs](https://phantombuster.atlassian.net/browse/CP-170)

### CODE REVIEW

**Review by Oxygen66 — APPROVED** [review]
Code LGTM 👍

File: src/contentscript/handlers/phantombuster.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import { FromBackgroundRuntimeMessages } from "../../shared/messages"
 import { Handler } from "./handler"
+import {version} from '../../../manifest.json'
```
Comment (RiahiKarim) [inline src/contentscript/handlers/phantombuster.ts:3]:
```suggestion
import { version } from '../../../manifest.json'
```

**Review by RiahiKarim — APPROVED** [review]
LGTM 💯

**Review by Oxygen66 — APPROVED** [review]
LGTM 👍

**Pull request merged**